tests/01_03_06-2wp_no_locking_cap.js

```diff
@@ -1,116 +1,79 @@
-      await btcClient.sendToAddress(addresses.btc, INITIAL_BTC_BALANCE);
-      await btcClient.generate(1);
-      await test.assertBitcoinBalance(addresses.btc, INITIAL_BTC_BALANCE, "Wrong initial BTC balance");
-      await wait(1000);
-      await test.assertLock(addresses, [{ address: federationAddress, amount: INITIAL_RSK_BALANCE }]);
+    const federationAddressBalanceInitial = Number(await btcTxHelper.getAddressBalance(federationAddress));
 
-      var lockedFedBalance = Number((await btcClient.getAddressBalance(federationAddress))[federationAddress]);
-      expect(lockedFedBalance).to.be.greaterThan(bitcoin.btcToSatoshis(1000) + initialFedBalance);
+    await btcTxHelper.fundAddress(btcAddressInfo.address, 3 * MINIMUM_PEGIN_VALUE_IN_BTC + btcTxHelper.getFee());
+    
+    const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, btcAddressInfo, 3 * MINIMUM_PEGIN_VALUE_IN_BTC);
+    await ensurePeginIsRegistered(rskTxHelper, btcPeginTxHash);
+    const recipientRskAddressBalanceAfterPegin = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address))
+    expect(Number(recipientRskAddressBalanceAfterPegin)).to.be.equal(Number(3 * MINIMUM_PEGIN_VALUE_IN_WEIS));
 
-      await rskClient.eth.personal.unlockAccount(addresses.rsk, '');
+    await sendTxToBridge(rskTxHelper, MINIMUM_PEGIN_VALUE_IN_BTC, recipientRskAddressInfo.address);
+    await rskUtils.triggerRelease(rskTxHelpers, btcTxHelper);
 
-      //Need to call sendTx in order to make it always with value < 1000, otherwise it fails
-      await rskClient.rsk.sendTx({
-        from: addresses.rsk,
-        to: rsk.getBridgeAddress(),
-        value: WEIS_TO_RELEASE,
-        gasPrice: TO_BRIDGE_GAS_PRICE
-      }, rskClient.evm.mine);
+    const federationAddressBalanceAfterFirstPegout = Number(await btcTxHelper.getAddressBalance(federationAddress));
+    expect(Number(federationAddressBalanceAfterFirstPegout)).to.be.equal(Number(federationAddressBalanceInitial + 2 * MINIMUM_PEGIN_VALUE_IN_BTC));
 
-      //Call release after sendTx so requests are released. Goes one by one
-      await rskUtils.triggerRelease(rskTxHelpers, btcTxHelper);
+    const senderAddressBalanceAfterFirstPegout = Number(await btcTxHelper.getAddressBalance(btcAddressInfo.address));
+    expect(Number(senderAddressBalanceAfterFirstPegout)).to.be.above(MINIMUM_PEGIN_VALUE_IN_BTC - btcTxHelper.getFee()).and.below(MINIMUM_PEGIN_VALUE_IN_BTC);
 
-      await wait(500);
-      
-      var afterFirstReleaseFedBalance = Number((await btcClient.getAddressBalance(federationAddress))[federationAddress]);
-      expect(afterFirstReleaseFedBalance).to.be.at.most(INITIAL_BTC_BALANCE / 2 + initialFedBalance);
+    const recipientRskAddressAfterFirstPegout = Number(await rskTxHelper.getBalance(recipientRskAddressInfo.address))
+    expect(Number(recipientRskAddressAfterFirstPegout)).to.be.above(MINIMUM_PEGIN_VALUE_IN_WEIS - btcTxHelper.getFee()).and.below(2 * MINIMUM_PEGIN_VALUE_IN_WEIS);
```

Comment:
The fee reduced here should be the fee spent in RSK to send the transaction to the Bridge. It's not the same as btc fee

Reply:
yes i know. is there a way how to get a RSK fee? i'm using the btc fee just to be little bit more precise...

Reply:
discussed on the call -> adding new constants